store/src/types.rs

```diff
@@ -28,6 +28,16 @@ use libc::{ftruncate64, off64_t};
 use core::core::hash::Hash;
 use core::ser;
 
+macro_rules! read_bitmap {
```

Comment:
(personal bias against macros) Can we do this with a simple fn and avoid having to introduce a macro?